scripts/updateMV3Blocklists.mjs

```diff
@@ -0,0 +1,64 @@
+import { dirname } from 'path'
+import { mkdirSync, writeFileSync } from 'fs'
+
+import { readEtags, writeEtags } from './utils.mjs'
+
+const currentTdsUrl = 'https://staticcdn.duckduckgo.com/trackerblocking/v6/current/extension-mv3-tds.json'
+const fallbackTdsUrl = 'https://staticcdn.duckduckgo.com/trackerblocking/v6/fallback/extension-mv3-tds.json'
+const currentTdsPath = './build/.intermediates/current-mv3-tds.json'
+const fallbackTdsPath = './build/.intermediates/fallback-mv3-tds.json'
+const currentTdsEtagKey = 'current-mv3-tds-etag'
+const fallbackTdsEtagKey = 'fallback-mv3-tds-etag'
```

Comment:
This seems like a lot of duplication. Couldn't we have these args provided via the command line and then run the script twice (once for `current` then once for `fallback`)?

Reply:
We could run the script twice to update the two files like you describe, or we could do it like this. I don't think it makes too much difference, but I went with this way for a couple of reasons:
1. If the update for one blocklist fails, we're not updating the etag for the other one.
2. We're not reading and writing to the etag.json file extra times.

Reply:
Making this atomic makes sense, but it would be nice if the file was more extendable, if we wanted to add more blocklists here. e.g. we could specify an array of lists at the top and fetch them all:
```javascript
const lists = [{
  url: 'https://staticcdn.duckduckgo.com/trackerblocking/v6/current/extension-mv3-tds.json',
  downloadPath: './build/.intermediates/current-mv3-tds.json',
  etagKey: 'current-mv3-tds-etag' 
}, {
 ...
}]
```

Reply:
Done.